packages/yew-router/src/lib.rs

```diff
@@ -78,16 +78,19 @@ pub mod switch;
 pub mod utils;
 
 pub use routable::{AnyRoute, Routable};
-pub use router::{BrowserRouter, HashRouter, Router};
+pub use router::Router;
+#[cfg(not(target_os = "wasi"))]
```

Comment:
Do we need these anymore now that gloo has been updated?

Reply:
> Do we need these anymore now that gloo has been updated?

Done